src/convertToObject.js

```diff
@@ -6,7 +6,29 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const stylesObject = {};
+
+  const rules = sourceString.split(';');
+
+  rules.forEach((rule) => {
+    if (!rule.trim()) {
+      return;
+    }
+
+    const [property, value] = rule.split(':');
+
+    if (!property || !value) {
+      return;
+    }
+
+    const cleanProperty = property.trim();
+
+    if (cleanProperty && value) {
```

Comment:
The check for 'cleanProperty && value' is unnecessary because you have already ensured that both property and value exist and are not empty in the previous checks.